modules/fleet-membership/outputs.tf

```diff
@@ -23,3 +23,14 @@ output "wait" {
   description = "An output to use when you want to depend on registration finishing"
   value       = var.enable_fleet_registration ? google_gke_hub_membership.primary[0].membership_id : local.gke_hub_membership_name
 }
+
+output "project_id" {
```

Comment:
Didn't our docs update for this change?

Reply:
Ah likely https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/issues/1817. I see in the run logs

```
Checking submodule's files generation
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=.git' /workspace/modules/fleet-membership/README.md /tmp/tmp.6g7TEY6pWv/workspace/modules/fleet-membership/README.md
39a40,41
> | location | The location of the hub membership. |
> | project\_id | The project of the hub membership. |
Error: submodule's files generation has not been run, please run the
'make build' command and commit changes
/usr/local/bin/task_helper_functions.sh: line 30: DELETE_AT_EXIT: readonly variable
Checking for documentation generation
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=autogen' '--exclude=*.tfvars' '--exclude=*metadata.yaml' /workspace/modules/fleet-membership/README.md /tmp/tmp.XkkMOPexjn/generate_docs/workspace/modules/fleet-membership/README.md
39a40,41
> | location | The location of the hub membership. |
> | project\_id | The project of the hub membership. |
Error: Documentation generation has not been run, please run the
'make docker_generate_docs' command and commit the above changes.
Checking for trailing whitespace
```

Reply:
Good Catch! Yeah, this is on old dev-tools.  Might make sense to do an incremental update just to get that fix.